travvy/src/components/Onboarding.vue

```diff
@@ -119,7 +119,21 @@ export default {
       console.log(this.instagram_username)
       console.log(this.preferred_activity_type)
       console.log(this.errors)
-
+      // sets the activity id based on the activity type
+      // recommends attractions based on activity id
+         if(this.preferred_activity_type == "Active"){
+            this.activity_id = 111
+        } else if (this.preferred_activity_type == "Food"){
+            this.activity_id = 112
+        }else if (this.preferred_activity_type == "Entertainment"){
+           this.activity_id = 113
+        } else if (this.preferred_activity_type == "Historical"){
+            this.activity_id = 114
+        } else if (this.preferred_activity_type == "Family"){
+            this.activity_id = 115
+        } else if (this.preferred_activity_type == "Nature"){
+            this.activity_id = 116
+        }
```

Comment:
Is there a reason we are using integer values here instead of text for activity.id as well?

Reply:
This is how they were structured in the database!